content/sensu-go/5.21/api/users.md

```diff
@@ -257,11 +319,12 @@ HTTP/1.1 201 Created
 ----------------|------
 description     | Updates the password for the specified Sensu user.
 example URL     | http://hostname:8080/api/core/v2/users/alice/password
-payload parameters | Required: `username` (string; the `username` for the Sensu user) and `password` (string; the user's new password).
+payload parameters | Required: `username` (string; the `username` for the Sensu user) and the `password` (string; the user's current password in cleartext).
```

Comment:
@palourde should this match the parameters description for  the`reset_password` endpoint?

```
Required: <ul><li>`username`: string; the username for the Sensu user</li><li>`password`: string; the user's current password in cleartext</li><li>`password_hash`: string; the user's hashed password via [bcrypt][3]</li></ul> 
```

Reply:
For that endpoint, you could provide either one.

Reply:
wait -- in https://github.com/sensu/sensu-docs/issues/2563#issuecomment-647741334 it says they need to provide password *and* password_hash